src/azure/cli/commands/arm.py

```diff
@@ -160,11 +160,22 @@ def required_values_validator(namespace):
 
 APPLICATION.register(APPLICATION.COMMAND_TABLE_LOADED, add_id_parameters)
 
-def register_generic_update(name, getter, setter, factory=None, setter_arg_name='parameters', #pylint:disable=too-many-arguments
-                            simple_output_query=None, custom_handlers=None):
+def _get_child_property(parent, collection_name, item_name):
+    items = getattr(parent, collection_name)
+    result = next((x for x in items if x.name.lower() == item_name.lower()), None)
```

Comment:
I changed it to be like the 'setter_params_arg_name' so we can override it.

---
In reply to: [76099476](https://github.com/Azure/azure-cli/pull/735#discussion_r76099476) [](ancestors = 76099476,76079355,75969337)